tracer/src/Datadog.Trace.Tools.Runner/GacInstallCommand.cs

```diff
@@ -46,17 +46,18 @@ private void Execute(InvocationContext context)
             return;
         }
 
-        using var container = NativeMethods.CreateAssemblyCache();
-        var hr = container.AssemblyCache.InstallAssembly(AssemblyCacheInstallFlags.None, assemblyPath, IntPtr.Zero);
-        if (hr == 0)
+        using var gacMethods = GacNativeMethods.Create();
+        var assemblyCache = gacMethods.CreateAssemblyCache();
+        var hr = assemblyCache.InstallAssembly(AssemblyCacheInstallFlags.IASSEMBLYCACHE_INSTALL_FLAG_REFRESH, assemblyPath, IntPtr.Zero);
```

Comment:
Is `IASSEMBLYCACHE_INSTALL_FLAG_REFRESH` really what we want? 🤔 Doesn't that try to _replace_ the version in the GAC? Don't we need to install them side-by-side?

Reply:
I wonder how the fleet installer gacinstall would interact with this tbh. That's something we should probably test (manually), even though I don't think it's an expected scenario 🤔